includes/enqueue-scripts.php

```diff
@@ -82,19 +82,15 @@ function edac_enqueue_scripts( $mode = '' ) {
 	);
 
 	if ( '' === $mode ) {
-		if ( ( current_user_can( 'edit_post', $post_id ) && ! is_customize_preview() )
-		) {
-			$mode = 'ui';
-		} else {
+		if ( ! current_user_can( 'edit_post', $post_id ) || is_customize_preview() ) {
 			return;
 		}
+		$mode = 'ui';
```

Comment:
In this case, reversing the logic of the condition allows us to bail early when needed.